src/client/pages/CountryHome/FraHome/hooks/useSections.tsx

```diff
@@ -19,21 +21,21 @@ type Section = {
   component: React.FC
 }
 
-export const useSections = (): Array<Section> => {
+export const useSections = (): Array<Section> | null => {
   const user = useUser()
   const { countryIso } = useCountryRouteParams()
-  const assessment = useAssessment()
   const cycle = useCycle()
+  const loaded = useDashboardLoaded()
+  const dashboardItems = useDashboardItems()
+  const hasDashboardItems = !Objects.isEmpty(dashboardItems)
 
   return useMemo(() => {
     const sections: Array<Section> = []
 
+    if (!loaded) return null
```

Comment:
When we are still loading the dashboard (note we check for area level) we return null.

We check on areaLevel because there might be differences in Region (no dashboard) and Country (has dashboard) level dashboards